capnp/src/schema_capnp.rs

```diff
@@ -752,14 +752,14 @@ pub mod node {
         11 => <crate::schema_capnp::node::annotation::Owned as crate::introspect::Introspect>::introspect(),
         12 => <crate::struct_list::Owned<crate::schema_capnp::node::parameter::Owned> as crate::introspect::Introspect>::introspect(),
         13 => <bool as crate::introspect::Introspect>::introspect(),
-        _ => panic!("invalid field index {}", index),
+        _ => panic!("invalid field index {index}"),
```

Comment:
Does this change (and the others in the generated `_capnp.rs` files) mean that `regenerate-capnp-schema-code.sh` and `regenerate-rpc-schema-code.sh` no longer suffice for regenerating these files?

Reply:
Please check, all have linting issues with nightly.

Reply:
Since these files are used to generate Rust files in the target project, we need to mark them as exceptions in our project. This PR resolves that inconvenient issue.
Look [here](https://github.com/pactus-project/tanour/blob/fe293fb90d4f8889d7483453bf6e37956220c73a/.github/workflows/workflow.yml#L50)